pkg/virtctl/credentials/common/common.go

```diff
@@ -67,7 +68,24 @@ func GetSshKey(flags *SshCommandFlags) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return string(data), nil
+	if len(data) == 0 {
+		return "", fmt.Errorf("ssh public key file %s is empty", flags.SshPubKeyFile)
+	}
+
+	dataLines := strings.Split(string(data), "\n")
+
+	return strings.TrimSpace(dataLines[0]), nil
```

Comment:
I think it would be valid to have multiple keys in a single pub key file. This silently drops keys and only returns the first?